src/components/common/Actions.vue

```diff
@@ -4,7 +4,7 @@
     <div class="buttons-left">
       <!-- disable Save button for now -->
       <v-btn id="save-btn" large
-        :disabled="!isEntityType || isBusySaving"
+        :disabled="!isEntityType || isBusySaving || !isFilingValid"
```

Comment:
Previously we allowed saving even when the filing was invalid. Please check with Linda and/or Scott whether we want to make this change. (Personally I can't decide.)

Reply:
I will check that

Reply:
Scott confirmed that saving should be enabled for invalid filing. So removed this condition from save / save and resume